e2e-tests/experiments/functional/lvm-volume-resize/run_e2e_test.yml

```diff
@@ -0,0 +1,51 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: lvm-vol-resize-
+  namespace: e2e
+spec:
+  template:
+    metadata:
+      name: lvm-vol-resize
+      labels:
+        app: lvm-vol-resize
+
+    spec:
+      serviceAccountName: e2e
+      restartPolicy: Never
+      containers:
+      - name: ansibletest
+        image: openebs/ansible-runner:ci
```

Comment:
@w3aman  is this image name is correct?

Reply:
will update it to `openebs/lvm-localpv-e2e:ci`